content_templates/templates/common/instance_view_builder.html

```diff
@@ -0,0 +1,405 @@
+                    {{ build_lam_property_configuration(graph, item, level + 1) }}
+                {% endfor %}
+            {% endif %}
+        {% endif %}
+    {% endcall %}
+{% endmacro %}
+
+
+{% macro build_lam_classification_hint_list(graph, instance_uri, level) %}
+    <p class="debug">build_lam_property_configuration_list <strong>{{ instance_uri }}</strong></p>
+    {% set select_query =
+    "PREFIX sh: <http://www.w3.org/ns/shacl#>
+    PREFIX lam: <http://publications.europa.eu/ontology/lam-skos-ap#>
+
+    select ?classificationHint
+    FROM <"~graph~">
+    {
+      values ?instance {<"~instance_uri~">}
+      ?instance lam:classifyWith ?classificationHint .
+    }" %}
+    {% set content, error = from_endpoint(conf.default_endpoint).with_query(select_query).fetch_tabular() %}
+    {% call macros.render_fetch_results(content, error) %}
+        {% for row in content.itertuples() %}
+            {{ build_lam_classification_hint(graph, row.classificationHint, level) }}
+        {% endfor %}
+    {% endcall %}
+{% endmacro %}
+
+
+{% macro build_lam_classification_hint(graph, instance_uri, level) %}
+    <p class="debug">build_lam_classification_hint <strong>{{ instance_uri }}</strong></p>
+
+    {% set select_query =
+    "PREFIX sh: <http://www.w3.org/ns/shacl#>
+
+    select ?name
+    FROM <"~graph~">
+    {
+      values ?instance {<"~instance_uri~">}
+      ?instance sh:name ?name .
+    }" %}
+    {% set content, error = from_endpoint(conf.default_endpoint).with_query(select_query).fetch_tabular() %}
+    {% call macros.render_fetch_results(content, error) %}
+        {% if not content.empty %}
+            {% for row in content.itertuples() %}
+                <h{{ level }}>
+                    {{ row.name }}<sup title="concept">[concept]</sup>
+                </h{{ level }}>
+            {% endfor %}
+        {% endif %}
+    {% endcall %}
+    {% set select_query =
+    "PREFIX sh: <http://www.w3.org/ns/shacl#>
+    PREFIX lam: <http://publications.europa.eu/ontology/lam-skos-ap#>
+
+    select ?path ?flags ?pattern ?class ?value ?minCount ?maxCount ?configurations
+    FROM <"~graph~">
+    {
+      values ?instance {<"~instance_uri~">}
+      {
+        ?instance lam:path ?path .
+      }
+      union {
+        ?instance sh:flags ?flags
+      }
+      union {
+        ?instance sh:pattern ?pattern
+      }
+      union {
+        ?instance sh:class ?class
+      }
+      union {
+        ?instance sh:hasValue ?value
+      }
+      union {
+        ?instance sh:minCount ?minCount
+      }
+      union {
+        ?instance lam:hasAnnotationConfiguration ?configurations
+      }
+    }" %}
+    {% set content, error = from_endpoint(conf.default_endpoint).with_query(select_query).fetch_tabular() %}
+    {% call macros.render_fetch_results(content, error) %}
+        {% if not content.dropna(how="all").empty %}
+            {{ macros.render_description_item_local_iri (content, "path", "Path") }}
```

Comment:
we have to think of a more compact way of rendering the constraints; 
a new macro has to be designed to produce something in one line or a small/compact table;

The constraints on a given property are quite predictable: 
a) either a class restriction
b) a value restriction or 
c) cardinality restriction (Min..Max)